### PR TITLE
fix content not displayed

### PR DESCRIPTION
**Pull request by xieyanker**
image

### CODE REVIEW

**Comment by xieyanker** [comment]
https://deploy-preview-21175--kubernetes-io-master-staging.netlify.app/zh/docs/concepts/policy/resource-quotas/

File: content/zh/docs/concepts/policy/resource-quotas.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 approvers:
 - derekwaynecarr
 title: 资源配额
+content_template: templates/concept
+weight: 10
```
Comment (tanjunchen) [inline content/zh/docs/concepts/policy/resource-quotas.md:6]: /lgtm
Thanks

**Pull request merged**